lib/utils.go

```diff
@@ -1,3 +1,5 @@
+// +build e2e_test
```

Comment:
Probably `library_test.go` should also be under this tag, no? It should not even compile for tests without `utils.go`.

Reply:
It's not needed there because `*_test.go` files are skipped when running `go build`. They are only included when running tests.

Reply:
Right, but someone will eventually run `go test` and will get compilation error and will be confused. :)

Reply:
Ah right. But it will also be confusing when one will run `go test ./lib` and it will print that there are no tests while `_test.go` file is present and actually has some tests. 

However, that may be less confusing as it will have the line at the top with a build tag... I will add at.